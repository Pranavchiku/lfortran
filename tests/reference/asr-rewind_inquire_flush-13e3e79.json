{
    "basename": "asr-rewind_inquire_flush-13e3e79",
    "cmd": "lfortran --indent --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/rewind_inquire_flush.f90",
    "infile_hash": "8a55dc820362b5ab028d6f06d1edc7476b8b0f03a76e7d87655825ad",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-rewind_inquire_flush-13e3e79.stdout",
    "stdout_hash": "64c8e6dfcab5dd7eaa2beb41aa18b67e791ffa83f15ac7f8c427fb0e",
    "stderr": "asr-rewind_inquire_flush-13e3e79.stderr",
    "stderr_hash": "8d29636224336bc6e52b181d9deacbfaa555d7773c4ce19731b37dcc",
    "returncode": 0
}