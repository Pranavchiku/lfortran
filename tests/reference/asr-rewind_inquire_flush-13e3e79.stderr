warning: Implied save feature
 --> tests/../integration_tests/rewind_inquire_flush.f90:3:5
  |
3 |     integer :: ios, len, a, b
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
 --> tests/../integration_tests/rewind_inquire_flush.f90:3:5
  |
3 |     integer :: ios, len, a, b
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
 --> tests/../integration_tests/rewind_inquire_flush.f90:3:5
  |
3 |     integer :: ios, len, a, b
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
 --> tests/../integration_tests/rewind_inquire_flush.f90:3:5
  |
3 |     integer :: ios, len, a, b
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
 --> tests/../integration_tests/rewind_inquire_flush.f90:4:5
  |
4 |     character :: fm
  |     ^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
 --> tests/../integration_tests/rewind_inquire_flush.f90:5:5
  |
5 |     logical :: ext
  |     ^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables
